sigs/test_aulldiv.pat

```diff
@@ -0,0 +1,4 @@
+53568b4424180bc075188b4c24148b44241033d2f7f18bd88b44240cf7f18bd3 48 d0a0 0068 :0000 __aulldiv
+558bec83e4e083ec60c5fe7f4c2420c5fe7f0424ff742424ff742424ff74240c 08 6e24 0095 :0000 ___avx_u64div4@@64 ^0028 __aulldiv
+558bec83e4f083ec30660f6fd1660f73da04660f6fd8660f7ed0660f73db04f3 1b ed6c 0080 :0000 ___sse2_u64div2@@32 ^003b __aulldiv
+---
```

Comment:
this signature is derived from siglib, which we created.

TODO: add license.

Reply:
though, i dont think the FLIRT lib supports comments. so, will need to add that feature.